pr_check.sh

```diff
@@ -58,14 +58,26 @@ B64_PASS=$(oc get secret ${ENV_NAME}-keycloak -o json | jq '.data.password' | tr
 # These ENVs are populated in the Jenkins job by Vault secrets
 python tmp/keycloak.py $HAC_KC_SSO_URL $HAC_KC_USERNAME $HAC_KC_PASSWORD $B64_USER $B64_PASS $HAC_KC_REGISTRATION
 
+cd integration-tests
+npm install
+npm install typescript
```

Comment:
Recommend to add this to the package.json and package-lock.json so that we have a pinned version here.